File: Atrium/src/main/java/Slot.java
package main.java;

import java.time.LocalTime;

public class Slot {

    private int capacity;
    private LocalTime hour;
    private Day day;

    public Slot(int capacity, LocalTime hour, Day day) {
        this.hour = hour;
        this.capacity = capacity;
        this.day = day;
    }

}

File: atrium_frontend/src/app/classes/slot.ts
import { Day } from "./day";
import { Time } from "./time";


export class Slot {
    slotid?: number ;
    
    day: Day ;
    time: Time ;
    capacity: number;

    constructor(day: Day, time: Time, capacity: number) {
        this.slotid = Math.random();
        this.day = day;
        this.time = time;
        this.capacity = capacity;
      }
}